manifest.json

```diff
@@ -2,6 +2,7 @@
   "manifest_version": 2,
   "name": "Vimium",
   "version": "1.49",
+  "minimum_chrome_version": "41",
```

Comment:
`chrome.tabs.sendMessage` only supports messaging individual frames from Chrome 41. #1575 needs 41+ for the same reason.

> Is 40 not a bit too recent to be excluding?

According to [here](http://gs.statcounter.com/#desktop-browser_version-ww-monthly-201504-201504-bar), Chrome 41/42 have 77.8% of Chrome's market total, Chrome 40 has just 2.8%. Still, I'm happy to remove the minimum version and add a check around the version-sensitive code, if that would make you less uncomfortable.